chainer_setup_build.py

```diff
@@ -388,6 +421,10 @@ def cythonize(extensions, force=False, annotate=False, compiler_directives={}):
 
     cython_cmdbase.extend(['--fast-fail', '--verbose', '--cplus'])
     for ext in extensions:
+        if all(not _is_changed(f) for f in ext.sources) and only_changed:
```

Comment:
`not _is_changed(f) for f in ext.sources` return the generator.
And, `all` method performs short-circuit evaluation.
`[not _is_changed(f) for f in ext.sources]` return the list.
`_is_changed` is called for all elements of `ext.sources`.